.storybook/preview-head.html

```diff
@@ -1,7 +1,388 @@
+    --cui-fg-on-strong-disabled: rgba(255, 255, 255, 0.4);
+    /* Accent foregrounds */
+    --cui-fg-accent: #3064e3;
+    --cui-fg-accent-hovered: #1c51d3;
+    --cui-fg-accent-pressed: #10399e;
+    --cui-fg-accent-disabled: rgba(48, 100, 227, 0.4);
+    /* Success foregrounds */
+    --cui-fg-success: #018850;
+    --cui-fg-success-hovered: #007a4e;
+    --cui-fg-success-pressed: #016c26;
+    --cui-fg-success-disabled: rgba(1, 135, 48, 0.4);
+    /* Warning foregrounds */
+    --cui-fg-warning: #e27900;
+    --cui-fg-warning-hovered: #cc6d00;
+    --cui-fg-warning-pressed: #b25c00;
+    --cui-fg-warning-disabled: rgba(232, 124, 0, 0.4);
+    /* Danger foregrounds */
+    --cui-fg-danger: #de331d;
+    --cui-fg-danger-hovered: #bd2c19;
+    --cui-fg-danger-pressed: #9e2415;
+    --cui-fg-danger-disabled: rgba(222, 51, 29, 0.4);
+    /* Promo foregrounds */
+    --cui-fg-promo: #9e33e0;
+    --cui-fg-promo-hovered: #8a1ecc;
+    --cui-fg-promo-pressed: #7219a9;
+    --cui-fg-promo-disabled: rgba(149, 53, 208, 0.4);
+    /* Neutral borders */
+    --cui-border-normal: #cccccc;
+    --cui-border-normal-hovered: #999999;
+    --cui-border-normal-pressed: #666666;
+    --cui-border-normal-disabled: rgba(204, 204, 204, 0.4);
+    --cui-border-subtle: #e6e6e6;
+    --cui-border-subtle-hovered: #cccccc;
+    --cui-border-subtle-pressed: #999999;
+    --cui-border-subtle-disabled: rgba(230, 230, 230, 0.4);
+    --cui-border-divider: #cccccc;
+    --cui-border-divider-hovered: #999999;
+    --cui-border-divider-pressed: #666666;
+    --cui-border-divider-disabled: rgba(204, 204, 204, 0.4);
+    --cui-border-strong: #1a1a1a;
+    --cui-border-strong-hovered: #000000;
+    --cui-border-strong-pressed: #000000;
+    --cui-border-strong-disabled: rgba(0, 0, 0, 0.4);
+    /* Accent borders */
+    --cui-border-accent: #3064e3;
+    --cui-border-accent-hovered: #1c51d3;
+    --cui-border-accent-pressed: #10399e;
+    --cui-border-accent-disabled: rgba(48, 100, 227, 0.4);
+    /* Success borders */
+    --cui-border-success: #018850;
+    --cui-border-success-hovered: #007a4e;
+    --cui-border-success-pressed: #016c26;
+    --cui-border-success-disabled: rgba(1, 135, 48, 0.4);
+    /* Warning borders */
+    --cui-border-warning: #e87c00;
+    --cui-border-warning-hovered: #cc6d00;
+    --cui-border-warning-pressed: #b25c00;
+    --cui-border-warning-disabled: rgba(232, 124, 0, 0.4);
+    /* Danger borders */
+    --cui-border-danger: #de331d;
+    --cui-border-danger-hovered: #bd2c19;
+    --cui-border-danger-pressed: #9e2415;
+    --cui-border-danger-disabled: rgba(222, 51, 29, 0.4);
+    /* Promo borders */
+    --cui-border-promo: #9e33e0;
+    --cui-border-promo-hovered: #8a1ecc;
+    --cui-border-promo-pressed: #7219a9;
+    --cui-border-promo-disabled: rgba(149, 53, 208, 0.4);
+    /* Special colors */
+    --cui-bg-overlay: rgba(0, 0, 0, 0.4);
+    --cui-border-focus: #ebf4ff;
+  }
+
+  @media (prefers-color-mode: dark) {
```

Comment:
This is a pseudo dark theme for the time being. It only switches the normal foreground and background colors. This should be updated with the real dark theme once we have it.